src/gui/widgets/TempoSyncKnob.cpp

```diff
@@ -75,7 +75,7 @@ void TempoSyncKnob::modelChanged()
 	{
 		m_custom->setModel( &model()->m_custom );
 	}
-	connect( model(), SIGNAL(syncModeChanged(lmms::TempoSyncKnobModel::TempoSyncMode)),
+	connect( model(), SIGNAL(syncModeChanged(lmms::TempoSyncKnobModel::SyncMode)),
```

Comment:
I find this odd. This code seems to pass in a ``SyncMode`` (an ``enum class``) as the signal argument and the builds pass? ``SIGNAL`` is a macro though, so I'm not sure how this is being parsed, but I think ``connect(model(), &TempoSyncKnobModel::syncModeChanged, this, &TempoSyncKnob::updateDescAndIcon);`` works better here. This also utilizes function pointers, [which have benefits over the ``SIGNAL``/``SLOT`` macros](https://doc.qt.io/qt-5/signalsandslots.html#signals-and-slots-with-default-arguments).